### ISSUE TITLE
spelunker links with archived layers errors in not informative fashion

### ISSUE THREAD

**Issue by fcollman** [CAVEconnectome/nglui#63]
The parsing feature does not work for spelunker/mainline neuroglancer annotations, now that nglui is supporting creating these links and tags are available in spelunker, but there is no csv download feature this is an important feature to add.

minimal broken example
```
client = caveclient.CAVEclient('minnie65_phase3_v1')
state_json = client.state.get_state_json(5929020391686144)
state = nglui.parser.StateParser(state_json)
state.annotation_dataframe()
```

leads to 

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
File /opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/box/box.py:593, in box.box.Box.__getitem__()

KeyError: 'props'

The above exception was the direct cause of the following exception:

BoxKeyError                               Traceback (most recent call last)
Cell In[9], [line 4](vscode-notebook-cell:?execution_count=9&line=4)
      [2](vscode-notebook-cell:?execution_count=9&line=2) state_json = client.state.get_state_json(5929020391686144)
      [3](vscode-notebook-cell:?execution_count=9&line=3) state = nglui.parser.StateParser(state_json)
----> [4](vscode-notebook-cell:?execution_count=9&line=4) state.annotation_dataframe()

File /opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:984, in StateParser.annotation_dataframe(self, expand_tags, point_resolution, split_points, include_archived)
    [956](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:956) def annotation_dataframe(
    [957](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:957)     self,
    [958](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:958)     expand_tags: bool = False,
   (...)
    [961](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:961)     include_archived: bool = True,
    [962](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:962) ) -> pd.DataFrame:
    [963](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:963)     """Get a dataframe with all annotations across all annotation layers in the state.
    [964](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:964) 
    [965](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:965)     Parameters
   (...)
...
    [373](https://file+.vscode-resource.vscode-cdn.net/opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/nglui/parser/base.py:373) if group:

File /opt/anaconda3/envs/cavePy312/lib/python3.12/site-packages/box/box.py:620, in box.box.Box.__getitem__()

BoxKeyError: "'props'"
```

**Comment by fcollman** [comment]
I see looking into this that this is general already suppose to be supported and this example is exposing some kind of flaw in the logic.

**Comment by fcollman** [comment]
I see this link issue was related to its archived layers and this fixed the parsing

```
client = caveclient.CAVEclient('minnie65_phase3_v1')
state_json = client.state.get_state_json(6460912698392576)
state = nglui.parser.StateParser(state_json)
state.annotation_dataframe(include_archived=False, expand_tags=True)
```

I would say the error message given here was not as informative as it could have been though to help figure this out.

**Comment by ceesem** [comment]
Fixed in v3.7.2!
